src/bn_accounts.erl

```diff
@@ -14,8 +14,25 @@
 %%
 
 handle_rpc(<<"account_get">>, {Param}) ->
+    Chain = blockchain_worker:blockchain(),
+    {Height, Ledger} =
+        case ?jsonrpc_get_param(<<"height">>, Param, false) of
+            false ->
+                {bn_txns:follower_height(), blockchain:ledger(Chain)};
+            V ->
+                case blockchain:ledger_at(V, Chain) of
+                    {ok, OldLedger} ->
+                        {V, OldLedger};
+                    {error, height_too_old} ->
+                        {V,
+                            ?jsonrpc_error(
+                                {error, "height ~p is too old to get historic account data", [V]}
+                            )};
+                    {error, _} = Error ->
+                        {V, ?jsonrpc_error(Error)}
```

Comment:
Code block for above

Reply:
makes sense that it'd complain. I'd say let's merge this for now to unblock.